[BUG] Error creating bean with name 'userRolesController'

- org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userRolesController': Unsatisfied dependency expressed through field 'service'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userRolesServiceImpl': Unsatisfied dependency expressed through field 'repository'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userRolesRepository' defined in com.amplifier.repositories.UserRolesRepository defined in @EnableJpaRepositories declared on JPAConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)! Reason: Validation failed for query for method public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)!; nested exception is java.lang.IllegalArgumentException: Validation failed for query for method public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)!
- Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userRolesServiceImpl': Unsatisfied dependency expressed through field 'repository'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userRolesRepository' defined in com.amplifier.repositories.UserRolesRepository defined in @EnableJpaRepositories declared on JPAConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)! Reason: Validation failed for query for method public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)!; nested exception is java.lang.IllegalArgumentException: Validation failed for query for method public abstract com.amplifier.models.UserRole com.amplifier.repositories.UserRolesRepository.findById(int)!

--- a/src/main/java/com/amplifier/repositories/UserRolesRepository.java
+++ b/src/main/java/com/amplifier/repositories/UserRolesRepository.java
@@ -17,7 +17,7 @@ public interface UserRolesRepository extends JpaRepository<UserRole, Integer> {
   @Query(value = "SELECT * FROM user_roles", nativeQuery = true)
   public List<UserRole> findAll();
 
-  @Query(value = "SELECT * FROM user_roles WHERE id=?1")
+  @Query(value = "SELECT * FROM user_roles WHERE id=?1", nativeQuery = true)
   public UserRole findById(int id);
 
   @Query(value = "UDPATE user_roles SET role=?1 WHERE id = ?2", nativeQuery = true)

